CMakeLists.txt

```diff
@@ -165,7 +160,15 @@ endif()
 
 # Setup environment.
 set(ROOTTEST_ENV_PATH ${ROOT_BINDIR})
-set(ROOTTEST_ENV_PYTHONPATH ${ROOT_LIBRARY_DIR})
+if(MSVC)
+  if(NOT PYTHON_EXECUTABLE_Development_Main) # Should not happen!
+    # FIXME: Temporary workaround until standalone roottest is properly fixed
```

Comment:
If you would like to have this in to go on with your testing, then it is ok for me.

But note that even this workaround might not work, if `PYTHON_EXECUTABLE` as it was found by standalone roottest has a different value than what ROOT found for ${PYTHON_EXECUTABLE_Development_Main}`.

Reply:
Yes, I know, but good enough for now (since the standalone use is broken anyway). And I'll remove the workaround once this is properly fixed.